src/test/libcephfs/test.cc

```diff
@@ -984,12 +984,23 @@ TEST(LibCephFS, PreadvPwritev) {
   struct iovec iovin[2];
   struct iovec iovout[2];
   ssize_t nwritten, nread;
+  char *out0, *out1, *in0, *in1; 
 
+  out0 = new char[str0.size()];
+  out1 = new char[str1.size()];
+  in0 = new char[str0.size()];
```

Comment:
It seems like that the runtime-sized array is supported in C++14, which was released on December 15, 2014. In other words, it's a new feature in C++. We cannot assume the ceph code will always be compiled with a compiler supporting the new C++ standard. To play safe, I think it would be better to do it with new/delete.